src/home/getters/get-github-user.ts

```diff
@@ -41,6 +42,7 @@ async function getNewGitHubUser(providerToken: string | null): Promise<GitHubUse
     const response = (await octokit.request("GET /user")) as GitHubUserResponse;
     return response.data;
   } catch (error) {
+    genericErrorModal(`${error}`)
```

Comment:
We do not want this displayed on a `403` since there is already the `handleRateLimit` that I believe would also do some UI changes.

Reply:
Okay in that case this becomes redundant. I've gotten rid of it. Thanks!

Reply:
This is still not solved. You just added a semi-colon.

Reply:
Pushed the wrong commit! So sorry for the hassle. It's fixed now